releasetasks/templates/enUS.yml.tmpl

```diff
@@ -1,3 +1,66 @@
+{% for platform, platform_info in en_US_config["platforms"].iteritems() %}
+
+{% if push_to_candidates_enabled %}  # beetmover
+{% set buildername = "release-{}_{}_{}_en-US_beetmover_candidates".format(branch, product, platform) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: opt-linux64
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=6) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        payload:
+            maxRunTime: 7200
+            # TODO - create specific image for this
+            image: rail/python-test-runner
+            command:
+                - /bin/bash
+                - -c
+                - >
+                  wget -O mozharness.tar.bz2 https://hg.mozilla.org/{{ repo_path }}/archive/{{ revision }}.tar.bz2/testing/mozharness &&
+                  mkdir mozharness && tar xvfj mozharness.tar.bz2 -C mozharness --strip-components 3 && cd mozharness &&
+                  python scripts/release/beet_mover.py --template configs/beetmover/en_us.yml.tmpl --platform {{ buildbot2ftp(platform) }} --version {{ version }} --app-version {{ appVersion }} --locale en-US --taskid {{ platform_info['task_id'] }} --build-num build{{ buildNumber }}
+            env:
+                DUMMY_ENV_FOR_ENCRYPT: "fake"
```

Comment:
Is `DUMMY_ENV_FOR_ENCRYPT` used somewhere? Not sure what the purpose of this is...